server/protocols/webadmin/webadmin-core/src/main/java/org/apache/james/webadmin/tasks/TaskFactory.java

```diff
@@ -38,15 +35,13 @@
 import com.google.common.collect.ImmutableSet;
 
 import spark.Request;
-import spark.Response;
-import spark.Route;
 
-public class TaskFactory {
+public class TaskFactory implements TaskGenerator {
```

Comment:
why?

Reply:
Because this is the case (request ->Task) and allows some code genericity / sharing.